api/src/test/java/gov/cms/ab2d/api/controller/AdminAPIPdpClientTests.java

```diff
@@ -363,7 +363,7 @@ public void getClientNotFound() throws Exception {
     }
 
     private void setupClient(String clientId, boolean enabled) {
-        Contract contract = dataSetup.setupContract(ENABLE_DISABLE_CONTRACT);
+        Contract contract = dataSetup.setupContract(ENABLE_DISABLE_CONTRACT, null);
```

Comment:
Instant red flag, adding a parameter that is now **null** for many cases.  See comments on the setupContract() method for how to fix.  

This will also greatly reduce the number of changes in this PR.